Use python3 in the qt5 GUI for th pulseaudio equalizer

diff --git a/src/utils/qpaeq b/src/utils/qpaeq
index ac0daec..dd53a32 100755
--- a/src/utils/qpaeq
+++ b/src/utils/qpaeq
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 #    qpaeq is a equalizer interface for pulseaudio's equalizer sinks
 #    Copyright (C) 2009  Jason Newton <nevion@gmail.com
 #
